moped-database/migrations/1703025658864_insert_into_public_moped_tags/up.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE "public"."moped_tags" SET "name" = 'Arterial Management - Inspection' WHERE "slug" = 'arterial_management';
```

Comment:
would you also update the slug to be `arterial_management_inspection`? sorry for not mentioning that in the issue!